public/diagnostic.js

```diff
@@ -23,3 +23,26 @@
 //  -  Order 1 : 2 hats ($5 each)
 //  -  Order 2 : 1 desk lamp ($20 each)
 //  -  Order 3 : 3 hand towels ($8 each)
+
+
+
+const Cart =  Ember.Object.extend({
+  orders: [],
+  addToCart(item) {
+    this.orders.push(item);
+    return ("Added " + item.description + "to the cart.");
+  },
```

Comment:
We need to use getters and setters if we want to take advantage of Ember's benefits.  So instead of `this.orders.push(item)` it would be `this.get('orders').addObject(item)`.  Also, `addToCart(item) {}` isn't a function, you need `addToCart: function(item) {}`